docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -6,18 +6,16 @@ position: 15
 
 For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on the [Octopus Server](docs/infrastructure/workers/built-in-worker.md), you can select a Docker image to execute the step inside of.
 
-:::warning
-Execution containers for workers are not currently supported for Octopus Cloud [Dynamic Worker pools](/docs/infrastructure/workers/dynamic-worker-pools.md).
-The dynamic workers do not have Docker installed. This will be addressed in the near future. 
-:::
-
 When an execution container is configured for a step, Octopus will still connect to the worker machine via a [Tentacle or SSH](/docs/infrastructure/workers/index.md#register-a-worker-as-a-listening-tentacle). The difference is that the specified image will be run as a container and the step will be executed inside the container (using the [docker exec](https://docs.docker.com/engine/reference/commandline/exec/) command).
```

Comment:
Just checking: I don't think we actually use the `docker exec` command to launch things. My understanding is that we use Calamari as the entry point and it just runs whatever it runs. The `docker exec` command launches a new process on a currently-running container. Do we need to clarify this?

Reply:
Oh, right.  I'll change this now.